packages/card/index.js

```diff
@@ -1,93 +1,90 @@
-import { FabricWebComponent } from '../utils';
+import { html, css } from 'lit';
+import { FabricElement } from '../utils';
 import { card as c } from '@fabric-ds/component-classes';
-import { classNames } from '@chbphone55/classnames';
+import { fclasses } from '../utils';
 
-class FabricCard extends FabricWebComponent {
-  static get observedAttributes() {
-    return ['selected'];
-  }
+const keys = {
+  ENTER: 'Enter',
+  SPACE: ' ',
+};
 
-  attributeChangedCallback(name, _, newValue) {
-    switch (name) {
-      case 'selected':
-        this[name] = newValue == 'false' ? false : true;
-        break;
-      default:
-        this[name] = newValue;
+class FabricCard extends FabricElement {
+  static styles = css`
+    a::after {
```

Comment:
More great questions! Again, I'm not really sure what the answer is on this and would welcome thoughts from others. @pearofducks @benja @Skadefryd ?

Reply:
Thinking about it some more though, wouldn't you just not use a link inside and make the card clickable or vice versa? Or if you did make the link and card clickable then you would make them do different things?

Reply:
The link is supposed to cover the entire card with `inset-0`.

We wrap the most important text on the card in a link/button for accessibility, but then extend that link via CSS to the entire card.

Reply:
<img width="1904" alt="Screen Shot 2022-05-10 at 22 21 51" src="https://user-images.githubusercontent.com/736895/167715339-dac9970e-73dd-44e2-a6af-1c20fafc6a79.png">

Reply:
This doesn't entirely seem to work properly in any of the implementations with regards to tabbing to the card/anchor I don't think. Tabbing should result in the blue focus state surrounding the card rather than the anchor right?

![Screenshot 2022-05-11 at 12 17 44](https://user-images.githubusercontent.com/1177098/167827431-2ab394b9-a98d-442a-ae0b-aa2b45c68a16.png)

Reply:
In any case, whether its correct or not, I don't think it has anything to do with card itself. (more likely clickable) so I think its ok to look at this further later if needed and merge this PR

Reply:
Does it mean that `a::after` style should eventually be moved to a `f-clickable` component when it's created?